packages/sandcastle/scripts/buildGallery.js

```diff
@@ -126,11 +172,17 @@ export function buildGalleryList(galleryDirectory, includeDevelopment = true) {
 // if running the script directly using node
 /* global process */
 if (import.meta.url.endsWith(`${pathToFileURL(process.argv[1])}`)) {
-  const __dirname = dirname(fileURLToPath(import.meta.url));
   const defaultGalleryDirectory = join(__dirname, "../gallery");
-  const { output, hasErrors } = buildGalleryList(defaultGalleryDirectory);
-  console.log("processed", output.entries.length, "sandcastles");
-  if (hasErrors) {
-    exit(1);
-  }
+  buildGalleryList(defaultGalleryDirectory)
+    .then(({ output, hasErrors }) => {
```

Comment:
Use `async/await` and `try/catch` for consistency with the rest of our code.

Reply:
I wanted to but eslint was complaining about top level await. Updated the config to consider these files the way we do node scripts.